web/generated_assets.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !genassets
+// +build !genassets
+
+package web
```

Comment:
maybe it would be worth to add a comment explaining this file is generated and shouldn't be modified.

Reply:
like 
`// THIS FILE HAS BEEN GENERATED, DO NOT EDIT.`

Reply:
Yup, that's a good idea.